### PR TITLE
Add stdint.readme file for toolchains that dont support stdint.h

### PR DESCRIPTION
**Pull request by aggarw13**
Add a `stdint.readme` file that can be used for toolchains that do not support the `stdint.h` standard C header file

### CODE REVIEW

File: source/include/stdint.readme [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#ifndef _STDINT_H
+#define _STDINT_H
+
+/*******************************************************************************
+ * THIS IS NOT A FULL stdint.h IMPLEMENTATION - It only contains the definitions
+ * necessary to build the coreMQTT code.  It is provided to allow coreMQTT to be
+ * built using compilers that do not provide their own stdint.h definition.
```
Comment (muneebahmed10) [inline source/include/stdint.readme:7]: My bad, I overlooked it earlier
```suggestion
 * THIS IS NOT A FULL stdint.h IMPLEMENTATION - It only contains the definitions
 * necessary to build the library code.  It is provided to allow the library to
 * be built using compilers that do not provide their own stdint.h definition.
```

**Review by muneebahmed10 — DISMISSED** [review]

File: source/include/stdint.readme [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+typedef unsigned char        uint8_t;
+typedef short                int16_t;
+typedef unsigned short       uint16_t;
+typedef long                 int32_t;
+typedef unsigned long        uint32_t;
+typedef long long            int64_t;
+typedef unsigned long long   uint64_t;
+
+#define INT8_MAX      ( ( signed char ) 127 )
+#define UINT8_MAX     ( ( unsigned char ) ) 255
```
Comment (dan4thewin) [inline source/include/stdint.readme:29]: One of these things is not like the others.

**Review by dan4thewin — CHANGES_REQUESTED** [review]

File: source/include/stdint.readme (hunk 2 above)
Comment (aggarw13) [inline source/include/stdint.readme:29]:
```suggestion
#define UINT8_MAX     ( ( unsigned char ) 255 )
```

Comment (aggarw13) [reply source/include/stdint.readme]: fixed

**Review by dan4thewin — APPROVED** [review]

**Review by muneebahmed10 — APPROVED** [review]

**Pull request merged**